pkg/webui/console/components/webhook-form/index.js

```diff
@@ -363,12 +406,42 @@ export default class WebhookForm extends Component {
             sensitive
             code
           />
+          {this.state.hasAuthorization && (
+            <Notification content={m.hasAuthorization} small warning />
+          )}
```

Comment:
The warning is not needed, though it can still make sense to generally show a warning when using duplicate warnings.